examples/pghostname_zig/build.zig.zon

```diff
@@ -0,0 +1,13 @@
+.{
+    .name = "pg_audit_zig",
```

Comment:
oopsie

Reply:
Ah, good catch, I copied it from `char_count_zig`, fixed in both places.